.readthedocs.yml

```diff
@@ -0,0 +1,14 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
+
+python:
+   version: "3.8"
```

Comment:
Quick note. Version `3.8` requires the `build.image` to be `latest`: https://docs.readthedocs.io/en/stable/config-file/v2.html#build-image